_posts/zh/newsletters/2018-12-18-newsletter.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Bitcoin Optech Newsletter #26"
+permalink: /zh/newsletters/2018/12/18/
+name: 2018-12-18-newsletter-zh
+slug: 2018-12-18-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 描述了用于带宽高效集和解的新 libminisketch 库，链接到关于 Schnorr/Taproot 计划的电子邮件，并提到即将召开的 LN 协议规范会议。还包括过去一周来自流行的比特币基础设施项目的几个值得注意的代码更改。
+
+## 行动项
+
+- **!--help-test-bitcoin-core-0.17.1RC1-->帮助测试 Bitcoin Core 0.17.1RC1：** 该[维护版本][maintenance release]的第一个候选版本已被[上传][V0.17.1rc1]。企业和个人用户对守护程序和 GUI 的测试非常受欢迎，并有助于确保高质量的发布。
```

Comment:
I noticed this in previewing the webpage in my browser.

```suggestion
- **<!--help-test-bitcoin-core-0.17.1RC1-->帮助测试 Bitcoin Core 0.17.1RC1：** 该[维护版本][maintenance release]的第一个候选版本已被[上传][V0.17.1rc1]。企业和个人用户对守护程序和 GUI 的测试非常受欢迎，并有助于确保高质量的发布。
```

Reply:
Thank you very much for your thorough review!

I'm sincerely sorry for not noticing it during the review. It's indeed my mistake that even missing a "<" symbol still passed the test. I will continue to review diligently in the future.